test/authoritative_fields/test_device_field_lock.py

```
@@ -0,0 +1,468 @@
+        )
+        assert can_overwrite is False
+
+        if can_overwrite:
+            device_handler.updateDeviceColumn(test_mac, "devName", proposed_value)
+            device_handler.updateDeviceColumn(test_mac, "devNameSource", plugin_prefix)
+
+        device_data = device_handler.getDeviceData(test_mac)
+        assert device_data.get("devName") == "Original Name"
+        assert device_data.get("devNameSource") == "LOCKED"
+
+    def test_field_source_tracking(self, test_mac, auth_headers):
```

Ruff (0.14.13)

Unused method argument: `auth_headers`

(ARG002)